hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java

```diff
@@ -2873,7 +2873,9 @@ public boolean checkBlockReportLease(BlockReportContext context,
   public boolean processReport(final DatanodeID nodeID,
       final DatanodeStorage storage,
       final BlockListAsLongs newReport,
-      BlockReportContext context) throws IOException {
+      BlockReportContext context,
+      int totalReportNum,
+      int currentReportNum) throws IOException {
```

Comment:
a. Please add some Javadoc about added parameter.
b. Will this name be more readable?
totalReportNum  -> totalStorageReportsNum, 
currentReportNum -> storageReportIndex